lnbits/extensions/lndhub/views_api.py

```diff
@@ -175,6 +175,7 @@ async def lndhub_getuserinvoices(
         offset=offset,
         exclude_uncheckable=True,
     ):
+        WALLET = get_wallet_class()
```

Comment:
can this be moved outside the loop?
`WALLET = get_wallet_class()`

Reply:
04b67c35